configs/xtensa-espressif_esp32_zephyr-elf.config

```diff
@@ -0,0 +1,66 @@
+CT_CONFIG_VERSION="3"
+CT_EXPERIMENTAL=y
+CT_LOCAL_TARBALLS_DIR="${CT_PREFIX:-${HOME}/x-tools}/sources"
```

Comment:
Please remove this as per https://github.com/zephyrproject-rtos/sdk-ng/commit/006aef1658fd51999d482dd28642e9d834adde3a.